arctic/store/version_store.py

```diff
@@ -131,6 +142,10 @@ def list_symbols(self, all_symbols=False, snapshot=None, regex=None, **kwargs):
         query = {}
         if regex is not None:
             query['symbol'] = {'$regex': regex}
+        if metadata:
```

Comment:
Presumably this will only be used for backwards compatibility - so, if possible, don't expose this to users. (Anything on the API has to exist forever)